src/main/java/io/anserini/search/SearchSolr.java

```diff
@@ -198,11 +198,18 @@ public SearchSolr(Args args) throws IOException {
 
     SolrDocumentList results = null;
 
+    // Replace some characters in query which are special syntax in Solr query parser
```

Comment:
Why can't this be replaced with a single regular expression?

Reply:
Thanks for the advice! Updated.